docs/getting-started/my-account/README.md

```diff
@@ -122,32 +119,84 @@ Click the grey **Register device** button to register an authentication app. Ope
+Access tokens allow you to grant external applications access to resources in OpenProject. 
+
+![openproject_my_account_access_tokens](openproject_my_account_access_tokens.png)
 
+### API
+API tokens allow third-party applications to communicate with this OpenProject instance via REST APIs. If you have not yet created an API token, this list will be empty. You may need admin privileges to be able to create an API.
 
+You can enable an API token under *Administration -> API and webhooks*. LINK TO HOW ITS DONE IN SYSTEM ADMIN GUIDE.
+### iCalendar
+iCalendar tokens allow users to subscribe to OpenProject calendars and view up-to-date work package information from external clients. 
+This list will be empty if you have not yet subscribed to any calendars. Once you [subscribe to a calendar] (LINK TO ICAL DOCUMENTATION), a list of all the calendars that you have subscribed to will appear here. The name of the calendar is clickable and will lead you directly to the respective calendar in OpenProject. 
+
+![OpenProject calendar list under my account](openproject_my_account_access_tokens_calendar_list.png)
+
+You can delete an entry in the iCalendar list by clicking on the **Delete** icon. This will trigger a warning message asking you to confirm the decision to delete.  
+
+![OpenProject delete calendar in My Account](openproject_my_account_access_tokens_delete_calendar.png)
+
+You will then see a message informing you that the the token und the iCal URL are now invalid.
+
+![OpenProject calendar access token is invalid](openproject_my_account_access_tokens_calendar_invalid.png)
+
+### OAUTH
+
+OAuth tokens allow third-party applications to connect with this OpenProject instance, for example Nextcloud (see [here](https://www.openproject.org/docs/user-guide/nextcloud-integration/)how to set up Nextcloud integration.  OAuth tokens can be created under [*Administration-> Authentication*](https://www.openproject.org/docs/system-admin-guide/authentication/). 
+
+If no third-party application integration has been activated yet, this list will be empty. Please contact your administrator to help you set it up. Once an integration has been set up, you will see the details here and will be able to delete any OAuth tokens by clicking on the **Delete** icon. 
+
+![OpenProject OAuth tokens under My Account](openproject_my_account_access_tokens_oauth.png)
+
+
+
+### RSS
+
+RSS tokens allow users to keep up with the latest changes in this OpenProject instance via an external RSS reader.  You can only have one active RSS token. 
+
+Create by clicking the **+RSS token** button. This will create your token and trigger a message showing you the access token. 
+
+>  **Note**: You will only be able to see the RSS access token once, directly after you create it. Make sure to copy it. 
+
+![OpenProject RSS token](openproject_my_account_access_tokens_rss.png)
```

Comment:
@MayaBerd same here in terms of security. Do we need to blur the token?

Reply:
@marcalcobe I dont think its critical, but I have blurred it to be safe